src/views/v2/rsvp/actions/rsvp/going.php

```diff
@@ -25,6 +25,6 @@ class="tribe-common-c-btn tribe-tickets__rsvp-actions-button-going tribe-common-
 		type="submit"
 		<?php tribe_disabled( $must_login ); ?>
 	>
-		<?php esc_html_e( 'Going', 'event-tickets' ); ?>
+		<?php echo esc_html_x( 'Going', 'Button label.','event-tickets' ); ?>
```

Comment:
:no_entry_sign: No space found after comma in argument list

<sub>hash: 70585eba371c202de942766c6a387246</sub>